app/controllers/ems_common.rb

```diff
@@ -766,4 +799,19 @@ def show_list_link(ems, options = {})
                                     :action     => "show_list",
                                     :id         => ems.id))
   end
+
+  #
+  # Returns the path of the template that generates the provider specific part of the new and edit
+  # provider dialogs.
+  #
+  # @param type [String] The type of provider, for example `rhevm`.
+  # @return [String] The path of the template.
+  #
+  def properties_template(type)
+    template = "/#{type}/properties"
```

Comment:
maybe prefix with a provider? e.g. /providers/type/<action>

Reply:
In the next version of the pull request I am adding that `/providers` prefix.